links/src/utils/__tests__/generateLinks.test.ts

```diff
@@ -1,11 +1,50 @@
-import { generateLinks } from "../generateLinks";
+import { generateLinks, totree } from "../generateLinks";
 import { test } from "uvu";
 import * as assert from "uvu/assert";
 
-test("generateLinks:", () => {
+test("generateLinks: ", () => {
 	generateLinks().then((data) => {
 		assert.equal(data, []);
 	});
 });
 
+test("totree: empty array", () => {
+	let tree = totree([], 0);
+	assert.equal(tree, []);
+});
+
+test("totree: array with one child", () => {
+	let tree = totree(tree2, 75884025);
+	assert.equal(tree, tree2);
+});
+
+type type2 = {
```

Comment:
Moving your types to `links/src/utils/generateLinks.ts` means you can utilise them in your functions.

```jsx
type Tree = ...
type Node = ...

export const toTree = (arr: Node[]): Tree => {
...
}
```